spyro/domains/quadrature.py

```diff
@@ -38,120 +35,34 @@ def quadrature_rules(V):
 
     if (cell_geometry == quadrilateral) and ufl_method == "Q":  # noqa: F405
         # In this case, for the spectral element method we use GLL quadrature
-        qr_x = gauss_lobatto_legendre_cube_rule(
-            dimension=dimension, degree=degree
-        )
+        qr_x = {"scheme": "KMV", "degree": degree}
         qr_k = qr_x
-        qr_s = gauss_lobatto_legendre_cube_rule(
-            dimension=(dimension - 1), degree=degree
-        )
+        qr_s = qr_x
     # elif (cell_geometry == quadrilateral) and ufl_method == "DQ":
     #     # In this case, we use GL quadrature
-    #     qr_x = gauss_legendre_cube_rule(dimension=dimension, degree=degree)
+    #     qr_x = {"scheme": "KMV", "degree": degree}
```

Comment:
```suggestion
    #     qr_x = {"scheme": "default", "degree": 2*degree}
```

Reply:
So, both  quad cases (when `cell_geometry == quadrilateral and ufl_method == "Q"` and when
`cell_geometry == TensorProductCell(quadrilateral, interval)` ) should use the GLL under-integrated quadrature, which is typically associated with Spectral Element Method (SEM) elements (i.e., GLL quadrature points collocated with Lagrange basis function nodes).

What is the default behavior at the moment?
I was following the fiat and finat calls from the Firedrake jupyter notebook:
https://github.com/firedrakeproject/firedrake/blob/release/docs/notebooks/10-sum-factorisation.ipynb